__tests__/EditHistoryTitleCollapsedActions.test.js

```diff
@@ -0,0 +1,39 @@
+jest.mock('app/services/Firebase', () => {});
+import thunk from 'redux-thunk';
```

Comment:
- [ ] Not on this file, but also test for the screen changes edit_quantifier and edit_collapsed_metric